opencti-platform/opencti-front/src/private/components/settings/settings_messages/SettingsMessageForm.tsx

```diff
@@ -0,0 +1,169 @@
+  ) {
+    settingsEdit(id: $id) {
+      editMessage(input: $input) {
+        messages {
+          ...SettingsMessagesLine_settingsMessage
+        }
+      }
+    }
+  }
+`;
+
+const messageValidation = () => Yup.object().shape({
+  message: Yup.string().required(),
+  activated: Yup.boolean().required(),
+  dismissible: Yup.boolean().required(),
+});
+
+type SettingsMessageInput = Partial<Pick<SettingsMessagesLine_settingsMessage$data, 'id' | 'activated' | 'message' | 'dismissible'>>;
+
+const SettingsMessageForm = ({
+  settingsId,
+  message,
+  handleClose,
+  creation = false,
+}: {
+  settingsId: string
+  message?: SettingsMessagesLine_settingsMessage$data
+  handleClose: () => void
+  creation?: boolean
+}) => {
+  const { t } = useFormatter();
+  const classes = useStyles();
+  const [commit] = useMutation(settingsMessageEditionPatch);
+
+  const onSubmit: FormikConfig<SettingsMessageInput>['onSubmit'] = (values, { setSubmitting }) => {
+    commit({
+      variables: {
+        id: settingsId,
+        input: values,
+      },
+      onCompleted: () => {
+        setSubmitting(false);
+        handleClose();
+      },
+    });
+  };
+
+  const initialValues = message ? {
+    id: message.id,
+    message: message.message,
+    activated: message.activated,
+    dismissible: message.dismissible,
+  } : {
+    message: '',
+    activated: false,
+    dismissible: false,
+  };
+
+  return (
+    <div>
+      <div className={classes.header}>
+        <IconButton
+          aria-label="Close"
+          onClick={handleClose}
+          size="large"
+          color="primary"
+        >
+          <Close fontSize="small" color="primary" />
+        </IconButton>
+        <Typography variant="h6" classes={{ root: classes.title }}>
+          {creation ? `${t('Create a message')}` : `${t('Update a message')}`}
+        </Typography>
+        <div className="clearfix" />
+      </div>
+      <div className={classes.container}>
+        <Formik<SettingsMessageInput>
+          enableReinitialize={true}
+          initialValues={initialValues}
+          validationSchema={messageValidation()}
+          onSubmit={onSubmit}
+        >
+          {({
+            submitForm,
+            isSubmitting,
+            isValid,
+          }) => (
+            <Form style={{ margin: '20px 0 20px 0' }}>
+              <Field
+                component={TextField}
+                variant="standard"
+                name="message"
+                label={t('Message')}
+                fullWidth={true}
+              />
+              <Field
+                component={SwitchField}
+                type="checkbox"
+                name="activated"
+                label={t('Activated')}
+                containerstyle={{ marginTop: 20 }}
+              />
+              <Field
+                component={SwitchField}
+                type="checkbox"
+                name="dismissible"
+                label={t('Dismissible')}
```

Comment:
Missing translation for `Dismissable`/ `Activated` (only french and spanish), `Message` (only french)

Reply:
I'm not very comfortable translating into Chinese and Japanese with google translate. We have opened discussions on this subject.

Any input @Kedae ?

Reply:
For now we google translate. That's the choice we made, we provide a default translation.

Reply:
we were just talking about translation yesterday with @yassine-ouaamou  and @sbocahu who suggested to use a special tool

Reply:
Yep, saw that, there is a page created. But until we do a move, we do the same as before :)